core/src/apps/bitcoin/common.py

```diff
@@ -43,8 +43,18 @@
     OutputScriptType.PAYTOP2SHWITNESS: InputScriptType.SPENDP2SHWITNESS,
     OutputScriptType.PAYTOWITNESS: InputScriptType.SPENDWITNESS,
 }
+
 INTERNAL_INPUT_SCRIPT_TYPES = tuple(CHANGE_OUTPUT_TO_INPUT_SCRIPT_TYPES.values())
+DECRED_INPUT_SCRIPT_TYPES = (
```

Comment:
these should be removed, see also comments in helpers.py

Reply:
https://github.com/trezor/trezor-firmware/pull/1249/commits/7581b3db49ea4d7762df6b63b8f3913d2074e3ba